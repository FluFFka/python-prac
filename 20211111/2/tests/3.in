""qweqwe
(1,2),(2,4),(10.123,20.246)
(-0.004, 4464.12345), (0.55, 4464.12345), (4465, 4463.12345)
