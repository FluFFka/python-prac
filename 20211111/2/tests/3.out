Invalid input
Not a triangle
0.27
